src/main/java/com/rackspace/salus/telemetry/ambassador/services/AgentHistoryService.java

```diff
@@ -73,6 +82,10 @@ public void addEnvoyConnectionClosedTime(String tenantId, String envoyId)  {
       final Instant connectionClosedTime = Instant.now();
       agentHistory.setDisconnectedAt(connectionClosedTime);
       agentHistoryRepository.save(agentHistory);
+    } else  {
+      log.warn("unable to find connection history with tenantId= {} and envoyId= {} ",tenantId, envoyId);
```

Comment:
Since you'll be in here for line below, we capitalize the first letter of log messages and no spaces in the `field={}` part 
```suggestion
      log.warn("Unable to find connection history with tenantId={} and envoyId={} ",tenantId, envoyId);
```